libexec/scoop-update.ps1

```diff
@@ -123,32 +123,42 @@ function update_scoop() {
         }
     }
 
-    if ((Get-LocalBucket) -notcontains 'main') {
-        info "The main bucket of Scoop has been separated to 'https://github.com/ScoopInstaller/Main'"
-        info "Adding main bucket..."
-        add_bucket 'main'
-    }
+    # This should have been deprecated after 2019-05-12
+    # if ((Get-LocalBucket) -notcontains 'main') {
+    #     info "The main bucket of Scoop has been separated to 'https://github.com/ScoopInstaller/Main'"
+    #     info "Adding main bucket..."
+    #     add_bucket 'main'
+    # }
 
     ensure_scoop_in_path
     shim "$currentdir\bin\scoop.ps1" $false
 
-    Get-LocalBucket | ForEach-Object {
-        write-host "Updating '$_' bucket..."
+    foreach ($bucket in Get-LocalBucket) {
+        Write-Host "Updating '$bucket' bucket..." -NoNewline
```

Comment:
@niheaven the `-NoNewline` flag is causing problems

![image](https://user-images.githubusercontent.com/46838874/149881121-d05b5cd1-f26e-4144-a49d-16d7866d5fed.png)

Also. I still feel the `Done.` output is unnecessary.

Reply:
Right, I'll do a quick hotfix.

Reply:
I noticed one more thing: we lost colored output due to the `Where-Object` in the end 😥

Reply:
Yes, I've noticed it too. Fixed in #4672